styles.css

```diff
@@ -0,0 +1,123 @@
+.page-container {
+    height: 100vh;
+}
+
+/*Nav bar*/
+.navbar {
+    padding: 1rem 3rem;
+    border-bottom: 1px solid var(--tertiary-color);
+}
+
+.nav-header {
+    flex: 1;
+    color: var(--primary-color);
+}
+
+.nav-icon {
+    color: var(--primary-color);
+}
+
+.nav-icon-text {
+    color: var(--primary-color);
+}
+
+
+/*main*/
+.homepage-text-img-container {
+    padding: 5rem 3rem;
+}
+
+.homepage-text {
+    color: var(--text-color1);
+    text-align: center;
+}
+
+.home-category-btn {
+    padding: 1rem;
+    width: fit-content;
+    text-align: center;
+    align-self: center;
+}
+
+.homepage-img {
+    height: 30rem;
+    width: 50rem;
+    border-radius: 2rem;
+}
+
+/*Categories*/
+.category-item {
+    cursor: pointer;
+    position: relative;
+}
+
+.category-item:hover {
+    opacity: 0.9;
+}
+
+.quiz-homepage-card {
+    width: 18rem;
+    height: 27rem;
+    box-shadow: var(--box-shadow);
+    transition: all 0.2s ease;
+    border-radius: 1rem;
+}
+
+.quiz-homepage-card:hover {
+    transform: scale(1.02);
+}
+
+.quiz-homepage-card-img-container {
+    width: 18rem;
+    height: 18rem;
+}
+
+.quiz-homepage-card-img {
+    width: 100%;
+    height: 100%;
+    border-top-left-radius: 1rem;
+    border-top-right-radius: 1rem;
+    object-fit: cover;
+}
+
+.homepage-category-textoverlay {
+    position: absolute;
+    width: 18rem;
+    height: 18rem;
+    color: red;
+    z-index: 2;
+}
+
+
+/*Footer*/
+.footer{
+    border: 1px solid var(--tertiary-color);;
+    z-index: 1;
+}
+
+.footer-social-icons {
+	margin: 1rem auto;
+    padding-left: 0;
```

Comment:
define padding 0 and margin 0 in your reset CSS or HTML tag so you don't have to use it elsewhere

Reply:
Yes I have already added the reset css. I forgot to remove the padding here. Thanks for pointing this out.